src/lib/commands/SendCommand.ts

```diff
@@ -78,9 +78,22 @@ export class SendCommand extends ExistingNewsLetterCommand {
     return ready;
   }
 
+  private extractEmail() {
+    let email = this.source;
+    if (typeof(email) === "string") {
+      const emailMatcher = new RegExp("^.*<(.+)>$");
```

Comment:
Could you add some tests for this logic? maybe extracting the helper function out of the instance.

Reply:
I could extract this into another class and write a test for it, but it seems kind of weird given your current tests. None of the commands are tested, for example. Standing up an entire test suite for `SendCommand` kinda seems like overkill to test this one function.

How would you like to proceed on this?

Reply:
You could add it as a static method and test it in a new test file SendCommand.test.ts. Until this change commands where mostly validation and integration code. Now that it modifies the data, it makes sense to have a test.

Reply:
OK, `extractEmail()` is now a public static function, with two passing tests.

Note that I had to install the pre-0.4 release deps, or the test suite was barfing with these errors (which seem unrelated to my tests):

```
jest SendCommand.test.ts
 FAIL  src/test/lib/commands/SendCommand.test.ts
  ● Test suite failed to run

    src/lib/commands/SendCommand.ts:1:8 - error TS1259: Module '"/home/hunmonk/git/newsletter-cli/node_modules/chalk/index"' can only be default-imported using the 'esModuleInterop' flag

    1 import chalk from "chalk";
             ~~~~~

      node_modules/chalk/index.d.ts:415:1
        415 export = chalk;
            ~~~~~~~~~~~~~~~
        This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        5.983 s
Ran all test suites matching /SendCommand.test.ts/i.
```